features/cross-document-view-transitions.yml

```
@@ -0,0 +1,9 @@
+name: Cross-document view transitions
+description: The `@view-transition` CSS rule sets whether a document opts-in to transitions from one document to another. Both the old and new document must opt-in for a transition to be triggered.
```

```suggestion
description: The `@view-transition` CSS rule sets whether a document opts-in to transitions between documents in a multi-page application. Both the old and new document must opt-in for a transition to be triggered.
```

Getting MPA in there for searchability.